hardware/RF-Test-Guide.md

```diff
@@ -0,0 +1,283 @@
+
+- Iteratively change component values until the S11 is acceptable.
+  a. The standard matching criterion is either -6 dB or -10 dB reflection across all over the frequency band.
+  b. When this goal is achieved, it is recommended to use the same values on a small batch of boards to make sure that this matching is acceptable across production tolerances.
+
+
+# 3. Frequency error and HFXO Capacitor Bank(CTune) calibration 
+
+## 3.1. Why is Ctune calibration important?
+It is recommended to calibrate the HFXO frequency for the devices to ensure minimum error of the radio carrier frequency. The link quality will be broken or degenerated if the frequency offset(error) is big between two communication nodes. The CTune is in charge of tuning the RF carrier frequency because the HFXO clock is used for the radio carier frequency synthesization.
+All Silicon Labs modules and radio boards are factory calibrated, and the CTune value is stored on the device information page. For other Silicon Labs radio boards, it is equipped with an external EEPROM which stores the CTune value. 
+For custom boards, the frequency error or crystal calibration should be performed especially for narrow bandwidth communication. It is preferred to calibrate 10-20 board's HFXO frequency per design, and get the average CTune value which can be used for the design. 
+
+## 3.2. Create Railtest tool for the test 
+First of all, it should generate a Railtest tool in Simplicity Studio for all the following RF test items.You can follow these steps to generate the test tools:
+- Connect customer board and debug adapter(BRD4001A mother board) to Simplicity Studio platform.
+- Select proper radio SoCs or module and related reference board 
+- Set debug mode to "Debug Out"
+- Select new project 
+- Select RAIL: RAILTEST 
+- Define a Railtest project name 
+- Select Finish
+- Configure the radio parameters 
+  - Modulation Type: FSK2
+  - Shaping Filter: Gaussian
+  - Shaping Filter Parameter: 0.5
+  - Bitrate [bps] / Deviation [Hz]: 125k/?, 500k/125k , 1M/250k, 2M/500k
+  - Modulation index: 0.5 based on BLE standard
+  - Checked the "Reconfigure for BER testing” box
+  - Everything else is default
+- Generate 
+- Build
+
+<div align="center">
+ <img src="files\HW-RF-Test-Guide\BER1.png">  
+</div>
+
+<div align="center">
+ <img src="files\HW-RF-Test-Guide\BER2.png">  
+</div>
+
+After build the Railtest tool, you can use the commander tool to program customer device, then the device will be ready for RF test.
+More information for Simplicity Studio usage, please visit the application note [AN0822](https://www.Silabs.com/documents/public/application-notes/AN0822-simplicity-studio-user-guide.pdf)
+
+## 3.3. Tune and measure
+The following strategy is proposed to determine the optimal Ctune value:
+- Performe a conducted measurement;
+- Connect the custom board's RF port to a Spectrum Analyzer(SA);
+- Use SA to measure the fundamental frequency as following picture shows:
```

Comment:
Might want to state that alternatively a frequency counter can be used for this measurement

Reply:
Added the content.